api/src/app.ts

```diff
@@ -105,6 +105,12 @@ export default async function createApp(): Promise<express.Application> {
 						// installations. Making this opt-in rather than opt-out is a little more
 						// friendly. Ref #10806
 						upgradeInsecureRequests: null,
+
+						// These are required for MapLibre
+						workerSrc: ["'self'", 'blob:'],
+						childSrc: ["'self'", 'blob:'],
+						imgSrc: ["'self'", 'data:', 'blob:'],
+						connectSrc: ["'self'", 'https://*'],
```

Comment:
Do we want to be explicit and define it as:

```js
connectSrc: ["'self'", 'https://*.tile.openstreetmap.org', 'https://*.tiles.mapbox.com', 'https://api.mapbox.com', 'https://events.mapbox.com']
```

instead? Not sure will there be any other URLs not covered by these (like how the maplibre docs didn't list out tile.openstreetmap.org), hence defaulted to `'https://*'` at the moment.

Reply:
No. The user can configured custom map sources in project settings, which may or may not come from these domains.